docs/date-picker/datepicker.md

```diff
@@ -1,8 +1,122 @@
+
+* *Selected and focused on different days* - 18th selected, 20th focused
+
+![Image of calendar view 4](./assets/datepicker/CalendarView-SelectedFocusedDifferent.png)
+
+* *Selected, focused and current on different days* - 18th selected, 20th current day, 11th focused
+
+![Image of calendar view 5](./assets/datepicker/CalendarView-SelectedCurrentFocusedDifferent.png)
+
+* *Changing the first day of the week* - In this example the first day of the week is set to Monday.
+
+![Image of calendar view 6](./assets/datepicker/CalendarView-firstdayofweek.png)
+
+* *Show prev/next days* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 7](./assets/datepicker/CalendarViewPrevNext.png)
+
+* *Disable Weekends* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 8](./assets/datepicker/CalendarViewDisabledWeekends.png)
+
+* *Disable specific dates* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 9](./assets/datepicker/CalendarView-DisableSpecificDates.png)
+
+* *Calendar View open - Bottom Left*
+
+![Image of calendar view 10](./assets/datepicker/CalendarView-OpenBL.png)
+
+* *Calendar View open - Bottom Right*
+
+![Image of calendar view 11](./assets/datepicker/CalendarView-OpenBR.png)
+
+* *Calendar View open - Top Left*
+
+![Image of calendar view 12](./assets/datepicker/CalendarView-OpenTL.png)
+
+* *Calendar View open - Top Right*
+
+![Image of calendar view 13](./assets/datepicker/CalendarView-OpenTR.png)
+
+## Properties
+
+DatePicker extends formInputProps
+
+| Name | Type | Default| Required | Description |
+| --- | --- | --- | --- | --- |
+| openOnFocus | boolean | false | no | Whether the Calendar View is opened when the input component gains focus. |
+| inputPlaceholder | string | empty string | no |Placeholder text for the input |
+| firstDayOfWeek | number | Defaults to 0 (Sunday), accepts 0 - 6 | no | define starting day of the week |
+| minDate | ISO 8601 date | empty date | no | Used to set the minimal date shown in the Calendar View and accepted in the input |
+| maxDate | ISO 8601 date | empty date | no | Used to set the maximum date shown in the Calendar View and accepted in the input |
+| inputFormat | string | browser implementation of javascript date object constructor | no | The format used to parse the input | 
+| disabledDates | Array [ISO 8601 date] | empty array | | List of ISO 8601 dates that are disabled (cannot be selected) in the Calendar View |
+| disableWeekends | boolean | false | no | Weekends cannot be selected in the Calendar View |
+| weekendDays | number | 1 | no |Defaults to [0,6], list of days set as weekend |
+| locale | string | user agent locale | no | Locale
+| dayLabels | Array[String] | english weekdays | no | Default in English, user may enter different labels (maybe can be done with code, in which case we change implementation - R&D)|
```

Comment:
do the "short versions" of dates have an array as well? (THU vs Thursday, but in hebrew ראשון vs. א) this may be a map that includes 2 arrays.

Reply:
Now as Array[{String, String}]